toolkits/global/packages/global-card/scss/10-settings/springer.scss

```diff
@@ -0,0 +1,17 @@
+/**
+ * Card
+ * Springer branding for component
+ */
+
+// Default setting
+@import 'default';
+
+// Springer overrides
+$card--padding-x: 0;
+$card--dark-background: #00285a;
+$card--dark-color: #fff;
+$card--title-font-weight: normal;
+$card--title-heading: h3;
```

Comment:
How does this `h3` work?

Reply:
@tulipdexter I'm not sure, I didn't write that.

Reply:
Looks like it's passed to a mixin:

```
@include u-heading($card--title-heading);
```

Reply:
Oh I see! Wow that really threw me. Thanks 👍